sdk/python/jobs/pipelines/pipeline_with_components_use_primitive_output/components_with_annotated.py

```diff
@@ -0,0 +1,60 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+from typing_extensions import Annotated
+from mldesigner import Input, Meta, Output, command_component
+from mldesigner.dsl import group
+
+
+@group
+class GroupOutputs:
+    output_int: Annotated[int, Meta(description="test annotation int")] = None
+    output_float: Annotated[float, Meta(description="test annotation float")] = None
+    output_str: Annotated[str, Meta(description="test annotation str")] = None
+    output_bool: Annotated[bool, Meta(description="test annotation bool")] = None
+
+    output_int2: int = None
+    output_float2: float = None
+    output_str2: str = None
+    output_bool2: bool = None
+
+    output_int3: Output(type="integer")
```

Comment:
maybe not expose such usage

Reply:
Based on the discussion, delete these codes and modify existing codes